libmodernize/main.py

```diff
@@ -80,9 +80,13 @@ def main(args=None):
     if not options.write and options.nobackups:
         parser.error("Can't use '-n' without '-w'.")
     if options.list_fixes:
-        print("Available transformations for the -f/--fix option:")
-        for fixname in sorted(avail_fixes):
-            print(fixname)
+        print("Available transformations for the -f/--fix and -x/--nofix options:")
+        if avail_fixes:
+            for fixname in sorted(avail_fixes):
+                print("    {}  ({})".format(fixname, fixname.split(".fix_", 1)[1]))
+        else:
+            print("    (None)")
```

Comment:
Can this ever happen? It looks like at least the 2to3 part of `avail_fixes` is hardcoded, so it should never be an empty set.

Reply:
Right, but this is listing *available* fixers, if I understand correctly - it's before any of the selection has been applied.

Reply:
Ah, I was looking at one of the other `(None)`s. It should be impossible but I added it as in the other areas for insurance against the unexpected (and future-proofing) out of habit. It could be deleted.

Reply:
OK. Let's either delete the branch, or put a comment mentioning that it should never happen, but it's just for insurance. Then I'm happy to merge this. :-)

Reply:
I'll remove the guard around it - there should always be some initial fixers possible, even if they all get cancelled out later.